mockwebserver/src/main/java/com/squareup/okhttp/mockwebserver/RecordedRequest.java

```diff
@@ -141,12 +143,14 @@ public int getSequenceNumber() {
     return sequenceNumber;
   }
 
-  /**
-   * Returns the connection's SSL protocol like {@code TLSv1}, {@code SSLv3},
-   * {@code NONE} or null if the connection doesn't use SSL.
-   */
+  /** @deprecated Use {@link #getTlsVersion()}. */
```

Comment:
Not sure how much we care about this. I'm going to be making other breaking changes in this API...